packages/theme/src/lib/legacy-theme.ts

```diff
@@ -508,6 +508,15 @@ export const legacyTheme = {
         maxHeight: '100px',
       },
     },
+    MuiCheckbox: {
+      styleOverrides: {
```

Comment:
Add disableRipple to default props and add the normal legacy focus styles - wonder if we can get away with just a border/boxshadow instead of both since the checkbox icon doesn't reach the bounds of the icon. 

Sidenote, i totally get why you were on the fence with the fontsize!!!

Reply:
Alright, I fixed the hover states. I also increased the fontSize to what amounts the small size from mui. 14px just felt too small.